spec/duration.html

```diff
@@ -1431,26 +1434,26 @@ <h1>
     <emu-clause id="sec-temporal-addduration" type="abstract operation">
       <h1>
         AddDuration (
```

Comment:
Are the BalanceDuration calls still infallible now?

Reply:
I think I have covered all of them now.